test/run-verify-host-user.sh

```diff
@@ -53,62 +53,62 @@ fi
+# (and thus are useful for reverse dependency testing and gating), and leave those which test cockpit-internal
+# functionality to upstream CI. We also need to leave out some which make too strict assumptions about the testbed.
 TESTS=""
 EXCLUDES=""
 RC=0
 if [ -n "$test_optional" ]; then
+    # don't include TestUpdates.testBasic: Testing Farm machines often have pending restarts/reboot
     TESTS="$TESTS
-         TestUpdates
-         TestAutoUpdates
-         TestStorage"
+         TestUpdates.testUpdateError
+         TestUpdates.testSecurityOnly
+         TestUpdates.testKpatch
+
+         TestAutoUpdates.testWithAvailableUpdates
+
+         TestStorage
+         "
 fi
 
 if [ -n "$test_basic" ]; then
-    # Testing Farm machines often have pending restarts/reboot
-    EXCLUDES="$EXCLUDES TestUpdates.testBasic TestUpdates.testFailServiceRestart"
+    TESTS="$TESTS
+        TestAccounts.testBasic
+        TestAccounts.testAccountLogs
 
-    # PCI devices list is not predictable
-    EXCLUDES="$EXCLUDES TestSystemInfo.testHardwareInfo"
+        TestJournal.testLogLevel
 
-    # No ABRT in CentOS/RHEL, thus not a test dependency
-    EXCLUDES="$EXCLUDES
-              TestJournal.testAbrtDelete
-              TestJournal.testAbrtReportCancel
-              TestJournal.testAbrtReport
-              TestJournal.testAbrtReportNoReportd
-              TestJournal.testAbrtSegv"
+        TestLogin.testBasic
+        TestLogin.testClientCertAuthentication
+        TestLogin.testSELinuxRestrictedUser
+        TestLogin.testTally
 
-    # FIXME: Often times out on at least c8s and f34 in TF
-    EXCLUDES="$EXCLUDES TestPages.testHistory"
+        TestServices.testBasic
+        TestServices.testBasicSession
 
-    TESTS="$TESTS
-        TestAccounts
-        TestBonding
+        TestSOS.testBasic
+
+        TestSystemInfo.testCPUSecurityMitigationsDetect
+
+        TestNetworkingBasic.testBasic
+        TestNetworkingVLAN
+        TestBonding.testBasic
         TestBridge
-        TestFirewall
-        TestKdump
-        TestJournal
-        TestLogin
-        TestNetworking
-        TestPackages
-        TestPages
-        TestServices
-        TestSOS
-        TestSystemInfo
         TestTeam
-        TestTerminal
+        TestNetworkingVLAN
+
+        TestFirewall.testFirewallPage
+        TestFirewall.testMultipleZones
+        TestFirewall.testAddServices
+
+        TestKdump
         TestTuned
         "
 
-    # HACK: check-sos fails 100% on Testing Farm Fedora 36 without any error message; works in local tmt VM
+    # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2058142
     if [ "$TEST_OS" = "fedora-36" ]; then
-        TESTS="${TESTS/TestSOS/}"
-    fi
-
-    # HACK: repeatedly fails on RHEL Testing Farm without error message, then corrupts VM
-    if [ "$TEST_OS" = "rhel-9-0" ]; then
-        EXCLUDES="$EXCLUDES TestPages.testBasic"
+        TESTS="${TESTS/TestSOS.testBasic/}"
```

Comment:
I don't really understand this: if we keep the excludes mechanism, why not use it for this?  Or dose it break if the test is explicitly listed in TESTS?

Reply:
No, we could use it. I am not entirely sure why we changed `$TESTS` for this back then (I kept the old code). But I'm happy to clean this up and use EXCLUDES, it's more consistent.

Reply:
Ah, now I know: We originally did this as we would otherwise have to mention all four TestSOS tests individually (--exclude specifices a single test by full name, by design). But now that we only run one, exclude is more appropriate.